### ISSUE TITLE
Provide vscode extension

### ISSUE THREAD

**Issue by trampster** [Microsoft/RoslynClrHeapAllocationAnalyzer#70]
I'm using dotnet core on linux with vscode to create a high performance timing critical application and am trying to reduce / eliminate my heap allocations in critical sections of my code.

Would be awesome if vscode was supported.

**Comment by edespong** [comment]
I think this should be possible to do once https://github.com/OmniSharp/omnisharp-vscode/issues/43 is all done.

**Comment by mjsabby** [comment]
I agree this should be possible. Let me know if anyone wants to take a swag at it.
